build/win/hyper

```diff
@@ -5,6 +5,9 @@ NAME="Hyper"
 HYPER_PATH="$(dirname "$(dirname "$(dirname "$(realpath "$0")")")")"
 ELECTRON="$HYPER_PATH/$NAME.exe"
 if grep -q Microsoft /proc/version; then
+	echo "Warning! Due to WSL limitations, you can use CLI commands here. Please use Hyper CLI on cmd, PowerShell or GitBash/CygWin."
```

Comment:
What does this mean? Do you mean "you cannot use Hyper CLI here"?

Reply:
yeah just missing the `'t`